src/css/style.css

```diff
@@ -0,0 +1,88 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  font-family: "Roboto", Helvetica, sans-serif;
```

Comment:
You better add font-family to html selector

Reply:
ok